cmd/githubRemoveValidate.go

```diff
@@ -41,7 +49,17 @@ var githubRemoveValidateCmd = &cobra.Command{
 
 		if repoMetaphor.GetName() == "metaphor" {
 			fmt.Println("metaphor should be not present")
-			return fmt.Errorf("error validating repo: %s ", repoGitops.GetName())
+			return fmt.Errorf("error validating repo: %s ", repoMetaphor.GetName())
+		}
+
+		if repoMetaphorGo.GetName() == "metaphor-go" {
```

Comment:
can `repoMetaphorGo` be another repo. name?

Reply:
? Not sure of the question, this is just integration tests, not sure the impact of the name.

Reply:
it could be empty, so the test it is to ensure it is empty. 
```
fmt.Println("metaphor should be not present") 
```

Reply:
we usually separate what is application code, and what is test.

it sounds strange to expect something different coming from `repoMetaphorGo.GetName` that is not the application name.